packages/x-components/tests/e2e/cucumber/partial-results/partial-results.spec.ts

```diff
@@ -1,20 +1,46 @@
 import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { createResultStub } from '../../../../src/__stubs__/results-stubs.factory';
+
+// Background
+Given('a results API with partial results', () => {
+  cy.intercept('https://api.empathy.co/search', req => {
+    req.reply({
+      results: [],
+      partialResults: [
+        {
+          query: 'verde azul',
+          results: [
+            createResultStub('Twister'),
+            createResultStub('Juego de Anillas Acuáticas Peces'),
+            createResultStub('Jurassic World Dinosaurio de Ataque Varios Modelos')
+          ],
+          totalResults: 9
+        },
+        {
+          query: 'lego verde',
+          results: [
+            createResultStub('LEGO Classic Ladrillos Creativos Verdes - 11007'),
+            createResultStub('LEGO Creator Grandes Dinosaurios -31058'),
+            createResultStub('LEGO My City Casa Familiar - 60291')
+          ],
+          totalResults: 6
+        }
+      ]
+    });
+  }).as('interceptedPartialResults');
+});
 
 // Scenario 1
 Given('no special config for partial-results view', () => {
-  cy.visit('test/partial-results');
+  cy.visit('test/partial-results?useMockedAdapter=true');
 });
 
 Then('at least {int} related results are displayed', (minResultsWithoutPartials: number) => {
   cy.getByDataTest('regular-result').should('have.length.at.least', minResultsWithoutPartials);
 });
 
 And('no partial results are displayed', () => {
```

Comment:
`partial results are not displayed` ?

Reply:
I would leave it as it is per consistency with the rest of steps across the project (`no identifier results are displayed`, `no query suggestion are displayed`, `no history queries are displayed`...)